activerecord/test/cases/tasks/sqlite_rake_test.rb

```diff
@@ -15,6 +15,9 @@ def setup
       File.stubs(:exist?).returns(false)
       ActiveRecord::Base.stubs(:connection).returns(@connection)
       ActiveRecord::Base.stubs(:establish_connection).returns(true)
+
+      @original_stdout = $stdout
```

Comment:
@Edouard-chin, yes i dont use `@original_stdout`. i defined this variable because of i think it is correctly -  save link to original `$stdout`, as you can see i override `$stdout` for tests in next line after defining `@original_stdout`.